regulation-worker/cmd/main_test.go

```diff
@@ -287,6 +411,69 @@ func getWorkspaceConfig(w http.ResponseWriter, r *http.Request) {
 	_, _ = w.Write(body)
 }
 
+func verifyBatchDelete() error {
```

Comment:
Is better for testing helper functions to use `t` for assertions instead of returning an error
```suggestion
func verifyBatchDelete(t *testing.T) {
    t.Helper()
```